meta-mender-core/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -1,4 +1,4 @@
-PACKAGECONFIG += "networkd resolved"
+PACKAGECONFIG_append = "networkd resolved"
```

Comment:
I used this to "validate":
[balister@thuvia build]$ bitbake -e systemd | grep ^PACKAGECONFIG
PACKAGECONFIG="xz                    ldconfig                    xkbcommon                    rfkill                    binfmt                    randomseed                    machined                    backlight                    vconsole                    quotacheck                    hostnamed                    myhostname sysusers                    hibernate                    timedated                    timesyncd                    localed                    ima                    smack                    logind                    firstboot                    utmp                    polkit  networkd resolved"

So I got 2 spaces after polkit. I see one space, I guess the LF adds another. At any rate, you are correct, consulting the manual says there should be a leading space to sbe safe from recipe changes. Good catch.